cluster/environments/azure-simple/variables.tf

```diff
@@ -40,6 +40,10 @@ variable "gitops_path" {
   default = ""
 }
 
+variable "gitops_url_branch" {
+  type = "string"
+}
```

Comment:
I would suggest adding a default value to keep configurable values in `.tfvars` at minimum.